extension.json

```diff
@@ -62,19 +62,25 @@
 		"Miraheze\\CreateWiki\\Maintenance\\RenameWiki": "maintenance/renameWiki.php"
 	},
 	"TestAutoloadNamespaces": {
+		"Miraheze\\CreateWiki\\Tests\\Unit\\": "tests/phpunit/unit",
 		"Miraheze\\CreateWiki\\Tests\\": "tests/phpunit/"
 	},
 	"Hooks": {
 		"BeforeCreateEchoEvent": [
 			"Miraheze\\CreateWiki\\Hooks::onBeforeCreateEchoEvent"
 		],
 		"LoadExtensionSchemaUpdates": [
-			"Miraheze\\CreateWiki\\Hooks::fnCreateWikiSchemaUpdates"
+			"main"
```

Comment:
Would the format of:
```json
{
	"handler": "main"
},
```
Not be better for better demonstration of using the handler? If not this is fine, just another option here.

Reply:
I couldn't find that syntax on [mediawikiwiki:Manual:Extension.json/Schema#Hooks](https://www.mediawiki.org/wiki/Manual:Extension.json/Schema#Hooks) and [mediawikiwiki:Manual:$wgHooks](https://www.mediawiki.org/wiki/Manual:$wgHooks), could I have a reference?

Reply:
I can't remember how I found it but that is the syntax I used for Cosmos.

Reply:
I just like the syntax better because it better explains usage of handler and makes it seem cleaner in my mind. But you to you here.

Reply:
As https://github.com/Universal-Omega/mediawiki-skins-Cosmos/blob/REL1_34/skin.json#L217-L227 doesn't seem to use it, I [searched using MediaWiki code search](https://codesearch.wmcloud.org/things/?q=%22Handler%22&i=fosho&files=extension%5C.json&excludeFiles=&repos=) and have found it is syntax. The wiki page should be improved...

Reply:
Why are you looking at REL1_34? The entire HookHandlers system wasn't added until MediaWiki 1.35 I believe. See the master branch for Cosmos. It does use it.

Reply:
Sorry, I didn't imagine it is maintained on Gerrit.